src/serializers/StackExchange.Redis.Extensions.System.Text.Json/StackExchange.Redis.Extensions.System.Text.Json.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Text.Json" Version="8.0.1" />
+    <PackageReference Include="System.Text.Json" Version="8.0.5" />
```

Comment:
Can be removed as provided by the aspnet framework

Reply:
No, it's different here because this lib is not tied to aspnetcore. It can be used on any host, such as a console app.

I've currently changed the codes to
```xml
    <PackageReference Include="System.Text.Json" Condition="'$(TargetFramework)' == 'net9.0'" Version="9.0.*" />
    <PackageReference Include="System.Text.Json" Condition="'$(TargetFramework)' == 'net8.0' OR '$(TargetFramework)' == 'netstandard2.1'" Version="8.0.*" />
```

And I can image it might become

```xml
    <PackageReference Include="System.Text.Json" Condition="'$(TargetFramework)' == 'net10.0'" Version="10.0.*" />
    <PackageReference Include="System.Text.Json" Condition="'$(TargetFramework)' == 'net9.0'" Version="9.0.*" />
    <PackageReference Include="System.Text.Json" Condition="'$(TargetFramework)' == 'net8.0' OR '$(TargetFramework)' == 'netstandard2.1'" Version="8.0.*" />
```
after release of .NET 10.

Reply:
Exactly, it's a console app that could be used everywhere, there is no reference to framework here

Reply:
Actually STJ is provided by the framework just like System.IO but was only added with net core 3: https://apisof.net/catalog/d6e30809188aaca14cbc44fd2000ab07

Reply:
@thompson-tomo Ah I see... you‘re right. 

Now I leave the explicit claim only when the TFM is `netstandard2.1`.